src/Rings/mpoly-affine-algebras.jl

```diff
@@ -1214,11 +1231,50 @@ julia> minimal_subalgebra_generators(V)
 ```
 """
 function minimal_subalgebra_generators(V::Vector{T}; check::Bool = true) where {T <: Union{MPolyDecRingElem, MPolyQuoRingElem{<: MPolyDecRingElem}}}
-  @req !isempty(V) "Input vector must not be empty"
-  I = ideal(parent(V[1]), [ zero(parent(V[1])) ])
-  return _minimal_subalgebra_generators_with_relations(V, I, check = check)[1]
+  return minimal_subalgebra_generators_with_relations(V; check=check)[1]
 end
 
+@doc raw"""
+    minimal_subalgebra_generators_with_relations(V::Vector{T}; check::Bool = true) where T <: Union{MPolyRingElem, MPolyQuoRingElem}
+
+Given a vector `V` of homogeneous elements of a positively graded multivariate
+polynomial ring, or of a quotient of such a ring, return a minimal subset `W` of
```

Comment:
Minimal in which sense? By inclusion (i.e. no further element can be dropped from `W`), or by size (which imply the former but be stronger in general)?

Reply:
It's of minimal cardinality, I clarified the doc string.